packages/tradershub/src/features/cfd/constants.tsx

```diff
@@ -72,46 +67,44 @@ export const MarketTypeDetails = (isEU?: boolean): TMarketTypeDetails => ({
     all: {
         description:
             'Trade swap-free CFDs on MT5 with forex, stocks, stock indices, commodities, cryptocurrencies, ETFs and synthetic indices.',
-        icon: <SwapFreeMT5Icon />,
-        iconWithWidth: (width: number) => <SwapFreeMT5Icon width={width} />,
+        icon: <PlatformIcon icon='SwapFree' />,
+        iconWithWidth: (width: number) => <PlatformIcon icon='SwapFree' width={width} />,
         title: 'Swap-Free',
     },
     financial: {
         description: isEU
             ? 'This MFSA-regulated account offers CFDs on derived and financial instruments.'
             : 'This account offers CFDs on financial instruments.',
-        icon: isEU ? <FinancialEUMT5Icon /> : <FinancialMT5Icon />,
-        iconWithWidth: isEU
-            ? (width: number) => <FinancialEUMT5Icon width={width} />
-            : (width: number) => <FinancialMT5Icon width={width} />,
+        icon: <PlatformIcon icon={isEU ? 'CFDs' : 'Financial'} />,
+        iconWithWidth: (width: number) => <PlatformIcon icon={isEU ? 'CFDs' : 'Financial'} width={width} />,
         title: isEU ? 'CFDs' : 'Financial',
     },
     synthetic: {
         description: 'This account offers CFDs on derived instruments.',
-        icon: <DerivedMT5Icon />,
-        iconWithWidth: (width: number) => <DerivedMT5Icon width={width} />,
+        icon: <PlatformIcon icon='Derived' />,
+        iconWithWidth: (width: number) => <PlatformIcon icon='Derived' width={width} />,
         title: 'Derived',
     },
 });
 
 export const PlatformDetails = {
     ctrader: {
-        icon: <CTraderIcon />,
-        iconWithWidth: (width: number) => <CTraderIcon width={width} />,
+        icon: () => <PlatformIcon icon='CTrader' />,
+        iconWithWidth: (width: number) => <PlatformIcon icon='CTrader' width={width} />,
```

Comment:
I think we don't need a different `iconWithWidth` rather we can just pass the default width to the same icon and make it simpler rather than two different icons

Reply:
I agree, wanted to change it but then it will will affect the current implementation behaviour. Will handle that in another PR.